horace_core/sqw/coord_transform/@aProjectionBase/aProjectionBase.m

```diff
@@ -903,8 +902,9 @@
             if ~isempty(obj.tmp_img_offset_holder_) && obj.alatt_defined && obj.angdeg_defined
                 img_offset_ = obj.tmp_img_offset_holder_(:);
                 obj.offset  = zeros(0,4); % nullify any previous offset
```

Comment:
Does `zeros(0, 4)` make sense? If it's `0` dimensional? Would `[]` or `zeros(1, 4)` not make more sense?

Reply:
I am following some previous wagualy specified convention. size of zeros(0,4) is [0,4] and size of [] is 0,0. Probably code should not rely on this, but it has been defined somewhere.